### PR TITLE
removed duplicate entries in ccode

### LINKED ISSUE #16598: remove duplicate entries for expm1 in ccode
There are duplicate lines in ccode: lines 64 and 65 define `'expm1': 'expm1',`.

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
#### References to other Issues or PRs

Fixes #16598
#### Brief description of what is fixed or changed
Removed duplicate entries of ``` 'expm1': 'expm1', ``` from ccode

#### Release Notes

NO ENTRY

### CODE REVIEW

**Review by oscargus — APPROVED** [review]
Looks good! I do not merge it now to save a bit of resources but can be merged at will when merging something else.

Thanks!

**Comment by Sc0rpi0n101** [comment]
@oscargus  
can you also review #16557  when you have time.

**Pull request merged**
